.github/workflows/run-workload.yaml

```diff
@@ -0,0 +1,78 @@
+name: run-workload
+description: Run a workload to create a new modeling job and associated task configs.
+
+on:
+  workflow_dispatch:
+    schedule:
+      - cron: '0 9 * * 3'
+    inputs:
```

Comment:
it looks like the scheduled run is inside the workflow dispatch? Am I reading that correctly?